spacy/cli/templates/quickstart_training.jinja

```diff
@@ -279,8 +279,12 @@ no_output_layer = false
 
 {% else -%}
 [components.textcat.model]
-@architectures = "spacy.TextCatCNN.v2"
+@architectures = "spacy.TextCatReduce.v1"
```

Comment:
This is used by the [generator](https://spacy.io/usage/training#quickstart) in the docs too, right? For the sake of compatibility with older spaCy versions, I think that we should consider keeping the old name here for a bit, as it's functionally the same anyway.

Reply:
Fixed.